Flatiron - Tuples chapter.md

```diff
@@ -0,0 +1,115 @@
+
+##What's a Tuple?
+Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
+
+**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
+```
+// Array
+let list = [ "Darth", "Ben", "Luke" ]
+print( list[1] )
+```
+Arrays are pretty useful but they're somewhat limited in that their contents can *only* be accessed via indices. 
+
+**Dictionaries** (as you might imagine by the name) help you manage data by mapping bits of it to keys, just like a language dictionary maps definitions to words.  The examples below use strings and arrays of strings but you could use almost anything (subject to certain rules that we'll cover in a later session).
+```
+// Dictionary
+let simpleDictionary = [ "Darth":"big baddie", "Ben":"mentor", "Luke":"protagonist" ]
+let richerDictionary = [
+	"Darth":[ "big baddie", "father of Luke" ],
+	"Ben":[ "mentor", "trainer of Darth" ],
+	"Luke":[ "protagonist", "son of Darth" ]
+	]
+print( richerDictionary[ "Darth" ] )
+[ "big baddie", "father of Luke" ]
+print( richerDictionary[ "Darth" ]![1] )
+"father of Luke"
+```
+Dicitionaries are very useful when you want a natural way of accessing contents and, crucially, you have a way of uniquely identifying each entry.
+
+**Classes and Structs** You may have already heard about these but they haven't yet been covered in class.  In brief, classes and structs are a powerful way to represent data because they guarantee the presence of specific properties (kind of like a dictionary which guarantees that there are entries for specific keys and, furthermore, can't have any more or fewer entries).  Classes and Structs are great to model specific types of data where you know exactly what should be present, such as a "Student" having a first name, last name, and array of test grades.
+
```

Comment:
I like your definition and the idea you're going for with this but if this was an actual lesson I would be inclined to move it more towards the bottom.

I prefer to structure the lesson in a way which is short with it's setup (or narrative), presenting the problem with which Tuples solve and diving RIGHT into tuples and breaking down bit by bit what a tuple represents, and how you create them using parenthesis, etc etc.